src/kms.ts

```diff
@@ -369,6 +381,67 @@ export class KmsClient {
     )
   }
 
+  /**
+   * Import a X509 certificate or a private key (both as DER encoded)
+   * @param {string} uniqueIdentifier  the unique identifier of the key
+   * @param {Uint8Array} derBytes the DER certificate/private key as bytes
+   * @param {string[]} tags potential list of tags
+   * @param {boolean} replaceExisting replace the existing object
+   * @param options Additional optional options
+   * @param {string} options.keyFormatType the key format type as specified in `object_data_structures.ts`
+   * @param {string} options.certificateIdentifier the private key id in order to create the link with the private key
+   * @returns {string}  the unique identifier of the key
+   */
+  public async importPrivateKey(
+    uniqueIdentifier: string,
+    derBytes: Uint8Array,
+    tags: string[] = [],
+    replaceExisting: boolean = false,
+    options: {
+      keyFormatType?: KeyFormatType
+      certificateIdentifier?: string
+    } = {},
+  ): Promise<string> {
+    const attributes = new Attributes()
+    attributes.objectType = "PrivateKey"
+    if (
+      options.certificateIdentifier !== undefined &&
+      options.certificateIdentifier.length > 0
+    ) {
+      attributes.link = [
+        new Link(LinkType.CertificateLink, options.certificateIdentifier),
+      ]
+    }
+    attributes.cryptographicLength = derBytes.length * 8
+
+    const privateKey = new PrivateKey(
+      new KeyBlock(
+        options.keyFormatType ?? KeyFormatType.ECPrivateKey,
```

Comment:
What happens if keyFormatType is a kind of public key ? No error should be raised? We will use 'importPrivateKey' to store public keys

Reply:
actually we won't be using the public key directly but only the X509 certificate